tests/plugins/test_resolve_remote_source.py

```diff
@@ -967,3 +967,27 @@ def test_generate_cachito_config_files_unknown_type(workflow):
 
     with pytest.raises(ValueError, match="Unknown cachito configuration file data type 'unknown'"):
         plugin.generate_cachito_config_files(dest_dir, cfg_files)
+
+
+def test_generate_cachito_config_create_parent_dir(workflow):
+    plugin = ResolveRemoteSourcePlugin(workflow)
+
+    dest_dir = Path(expected_build_dir(workflow))
+    cfg_files = [
+        {
+            "path": ".bundle/foo",
+            "type": CFG_TYPE_B64,
+            "content": base64.b64encode(b"contents")
+        }
+    ]
+
+    plugin.generate_cachito_config_files(dest_dir, cfg_files)
+
+
+def test_generate_cachito_config_parent_dir_already_exists(workflow):
```

Comment:
It might be best to add another config file to the test constant instead: https://github.com/containerbuildsystem/atomic-reactor/blob/e54f9d295656f7e09c73d33614391f2451e60af9/tests/plugins/test_resolve_remote_source.py#L214-L221

And update the places where `check_injected_files()` is used to check the file contents